src/reducers/editor.js

```diff
@@ -598,7 +598,7 @@ const fixStateReferences = (module, stateName, newName) => {
     }
     if(state.target_encounter === stateName){
       if(newName === null){
-        state.target_encounter = "" // this is a requried field
+        delete state.target_encounters
```

Comment:
Should this be pluralized? I didn't dig into this but at glance it looks like it's only `target_encounter` everywhere else

Reply:
Good catch.